Control-Tower Changes - Migrating from AWS API polling integrations to Metrics Streaming (#32)
aws-quickstart/quickstart-ct-newrelic-one: docs/partner_editable/faq_troubleshooting.adoc

*Q.* I included a list of AWS accounts during the deployment, but none of the accounts show up in New Relic's UI.

*A.* Ensure that your list includes only AWS account IDs that are managed by AWS Control Tower and that the account IDs are separated by commas. Also ensure that you deployed the stack in the same Region as your AWS Control Tower management account and that the New Relic NerdGraph user API key is correct. Lastly, inspect the Amazon CloudWatch logs for errors regarding the AWS Lambda function `NewRelicOnboardingFunction`.

> *Q.* How are Metrics from AWS Account sent to New Relic Platform.

Reviewer: ```suggestion
*Q.* How are metrics delivered to the New Relic platform?
```
